### ISSUE TITLE
Changing the changelog

### ISSUE THREAD

**Issue by mahrud** [Macaulay2/M2#3134]
I find the style of changes.m2 and the nodes documented by it very hard to use. For instance, each version has its own page, so often I'd have to open half a dozen different pages to look for something, at which point I usually give up and look at the source instead! Also, it's hard to track changes to a particular package or feature.

Curious what are people's thoughts about this.

**Comment by d-torrance** [comment]
I agree that it would be nice if the contents of `changes.m2` all appeared together in a single file.

**Comment by DanGrayson** [comment]
Will do!

**Comment by mahrud** [comment]
Will do what?

**Comment by DanGrayson** [comment]
Put all the changes in a single file.

**Comment by mahrud** [comment]
Maybe I should have been more clear. I was asking for everyone's opinions before any particular change is implemented.

For instance, I'd like every package (including Macaulay2Doc) to have a special page called "History" or "Changelog" which is linked in the header, and lists the updates to that package all in one page.

**Comment by DanGrayson** [comment]
Okay, I won't do it.

**Comment by d-torrance** [comment]
> **Written by Claude** (Claude Opus 5, via Claude Code), not by @d-torrance, whose account posted it -- please weigh it accordingly.

Closing as `wontfix` — not because the inconvenience isn't real, but because each of the concrete
proposals here has already been settled one way or the other.

**The single-file consolidation was offered and turned down.** @DanGrayson volunteered it twice in
this thread; the reply was that it wasn't what was being asked for. That leaves it declined by the
reporter rather than ignored by the maintainers.

**Per-package changelogs are the author's call, and the practice already exists** without needing
new infrastructure. Eleven distributed packages carry version-history prose in their documentation
today, several of them in the top-level node — `A1BrouwerDegrees`, `BettiCharacters`,
`HighestWeights`, `InvariantRing`, `Jets`, `LieAlgebraRepresentations`, `NeuralIdeals`, `Pullback`,
`RunExternalM2`, `TestIdeals`, `VectorFields`. Others keep the history in source comments. An author
who wants a History page for their package can write one now.

That leaves the per-version structure of `changes.m2` itself, which has been the status quo for
years and for which no replacement has been proposed that the reporter and the maintainers agree
on. If it is worth revisiting with a specific design, a new issue carrying that design would be a
better starting point than this thread.

<sub>Triaged in a sweep of the open issues; source of truth: [`bug-triage/issues.tsv`](https://github.com/d-torrance/M2/blob/bug-triage/bug-triage/issues.tsv).</sub>
